README.md

```diff
@@ -209,14 +207,31 @@ rows = [
   ['wrong column count']
 ]
 
-schema.convert(rows)
+schema.cast(rows)
```

Comment:
From my experience almost all interactions with schema related to casting go thru `schema.cast_row` - not sure this method needed at all. Also name could be confusing (`schema.cast` - cast what? schema? data?). WDYT?